pandas/core/generic.py

```diff
@@ -3753,9 +3755,17 @@ def sample(self, n=None, frac=None, replace=False, weights=None,
         -------
         A new object of same type as caller.
 
-        Examples
+        See Also
         --------
+        Series.sample : Returns a random sample of items
+            from an axis of object.
+        DataFrame.sample : Returns a random sample of items
+            from an axis of object.
+        Panel.sample : Returns a random sample of items
+            from an axis of object.
 
+        Examples
+        --------
```

Comment:
Just below this line would be a good place to mention "We use a random state for reproducible.

Do you have any interest in updating the examples to use non-random data?